src/cryptonote_basic/cryptonote_format_utils.h

```diff
@@ -139,6 +139,9 @@ namespace cryptonote
   unsigned int get_default_decimal_point();
   std::string get_unit(unsigned int decimal_point = -1);
   std::string print_money(uint64_t amount, unsigned int decimal_point = -1);
+#ifdef BOOST_MP_CPP_INT_HPP
+  std::string print_money(const boost::multiprecision::uint128_t &amount, unsigned int decimal_point = -1);
+#endif
```

Comment:
Why use this macro switch instead of just including ` <boost/multiprecision/cpp_int.hpp>` here? Having header inclusions for any types needed by a function declared in a header seems like a normal practice to me.

Reply:
That header looked like it was including a lot of stuff that's rare, so I thought I'd save the compile time hit since cryptonote_format_utils.h is included by pretty much everything.

Reply:
ok